KmsClient.js

```diff
@@ -701,3 +743,36 @@ function _getRootZcapId({keystoreId, keyId}) {
   }
   return `${ZCAP_ROOT_PREFIX}${encodeURIComponent(suffix)}`;
 }
+
+/**
+ * Creates and signs the http headers for zCap invocation.
+ *
+ * @param {object} options - Options hashmap.
+ * @param {string} options.url - zCap target url.
+ * @param {string} options.method - HTTP method ('get', 'post').
+ * @param {object} [options.headers] - HTTP headers.
+ * @param {object} [options.operation] - Optional JSON payload (for POSTs).
+ * @param {object} [options.capability] - Optional existing capability.
+ * @param {{sign: Function}} options.invocationSigner - Key Signer object.
+ * @param {string} options.capabilityAction - zCap action to perform.
```

Comment:
```suggestion
 * @param {string} options.capabilityAction - The zCap action to perform.
```

Reply:
@dlongley: -1 to that, I really don't like extraneous `The`, for param docstrings.

Reply:
(I did fix the capitalization, though.)